src/proxy.ts

```
@@ -168,6 +217,207 @@ class TranscriptionProxy {
+        logger.error(`Async transcription failed: ${data?.error || "unknown error"}`);
+        this.audioVisualizer.addLog(`Transcription failed: ${data?.error}`, "error");
+        break;
+
+      case "meeting.ended":
+        logger.info("Meeting has ended");
+        this.audioVisualizer.addLog("Meeting ended", "info");
+        break;
+
+      case "bot.status_change":
+        // status can be a string or an object with code property
+        const statusObj = data?.status;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)